File: ООП/Homework2/all_animals/animal.java
package all_animals;

public abstract class animal{

    private int height;
    private int weight;
    private String eyes_color;

    protected animal(int height, int weight, String eyes_color){
        this.height = height;
        this.weight = weight;
        this.eyes_color = eyes_color;
    }
    public abstract void voice();

    public String toString(){
        return String.format("%d \nРост: %d \nВес: %d \nЦвет глаз: %s", this.height, this.weight, this.eyes_color);
    }
}

File: ООП/Homework2/animals_PY/classes/animal.py
from abc import ABC, abstractclassmethod

class animal(ABC):
    def __init__(self, name, height, weight, coloreye):
        self.name = name
        self.height = height
        self.weight = weight
        self.coloreye = coloreye

    def __str__(self) -> str:
        return f"{self.name}\nРост: {self.height}\nВес: {self.weight}\nЦвет глаз: {self.coloreye}"
    
    def voice(self):
        return (f"{self.name}: ")